prow/apis/prowjobs/v1/types.go

```diff
@@ -373,6 +373,10 @@ type DecorationConfig struct {
 
 	// CensoringOptions exposes options for censoring output logs and artifacts.
 	CensoringOptions *CensoringOptions `json:"censoring_options,omitempty"`
+
+	// IgnoreInterrupts causes sidecar to ignore interrupts and
+	// hope that the test process exits cleanly before starting an upload.
+	IgnoreInterrupts *bool `json:"ignore_interrupts,omitempty"`
```

Comment:
I think we should rename this to something that indicates that this applies to the job result upload only and that interrupts will still be forwarded to the test process, e.g. `upload_ignores_interrupts`

Reply:
Updated it to `upload_ignores_interrupts`